app/admin/AdminLayoutClient.tsx

```diff
@@ -90,6 +90,15 @@ export default function AdminLayoutClient({
                   Simulator
                 </Link>
               </li>
+              <li>
+                <Link
+                  href="/admin/browser-sessions"
+                  className="block p-3 hover:bg-gray-200 hover:text-black rounded text-white"
+                  onClick={() => setSidebarOpen(false)}
+                >
+                  Browser Sessions
+                </Link>
+              </li>
```

Comment:
Delete this tab. We want to scope the browser sessions within an instance like we have now